content/en/blog/_posts/2020-12-16-third-party-device-metrics-hits-ga.md

```diff
@@ -0,0 +1,134 @@
+---
+ layout: blog
+ title: 'Third Party Device Metrics Hits G.A'
+ date: 2020-12-16
+ slug: third-party-device-metrics-hits-ga
```

Comment:
Netlify build is failing due to the error below. I believe it's complaining about the spaces.
```
1:13:22 PM: Error: Error building site: "/opt/build/repo/content/en/blog/_posts/2020-12-16-third-party-device-metrics-hits-ga.md:2:1": EOF looking for end YAML front matter delimiter
```

```suggestion
layout: blog
title: 'Third Party Device Metrics Hits G.A'
date: 2020-12-16
slug: third-party-device-metrics-hits-ga
```